src/__tests__/use-app/__fixtures__/custom-app-with-gip-custom-props/pages/ssg.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { sleep, PropsPrinter } from '../../../../__utils__/';
+
+export default function CustomAppWithGIPCustomProps_SSG({
+  ctx: _ctx,
+  ...props
+}) {
+  return (
+    <>
+      custom-app-with-gip-custom-props/ssg -
+      <PropsPrinter props={props} />
+    </>
+  );
+}
+
+export async function getStaticProps() {
+  await sleep(1);
+  return {
+    props: {
+      fromPage: true,
+      propNameCollision: 'from-page',
```

Comment:
Just asking: is `propNameCollision` relevant for the scope of this test?

Reply:
not relevant, removed